blocks/video-player-block/features/video-player/util/formatEmbedMarkup.js

```diff
@@ -0,0 +1,48 @@
+/**
```

Comment:
Is this the same as the Arc Themes Components - `formatPowaVideoEmbed` function? https://github.com/WPMedia/arc-themes-components/blob/arc-themes-release-version-2.00/src/utils/format-powa-video-embed/index.js

Reply:
Yep, I didn't see that... will update.